met-api/src/met_api/services/widget_documents_service.py

```diff
@@ -29,6 +29,7 @@ def get_documents_by_widget_id(widget_id):
     @staticmethod
     def _attach_file_nodes(docs, root):
         files = list(filter(lambda doc: doc.type == DocumentType.FILE.value, docs))
+        files.sort(key=lambda doc: doc.id)
```

Comment:
is there a requirement to sort by id?

Reply:
yes without sorting by id for the files, when there is multiple files without a folder they randomly change order after being edited.